pkgs/tools/security/ghidra/extensions.nix

```diff
@@ -29,4 +29,6 @@ lib.makeScope newScope (self: {
 
   sleighdevtools = self.callPackage ./extensions/sleighdevtools { inherit ghidra; };
 
+  wasm = self.callPackage ./extensions/wasm { inherit ghidra; };
+
```

Comment:
```suggestion
```

Reply:
Previous file had trailing newline, so I left it in.